### PR TITLE
test: resume task to fix unstable test case

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
part of https://github.com/pingcap/dm/issues/1790

### What is changed and how it works?
described in comment of test

### Check List 

Tests 

 - No code

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: tests/shardddl1/run.sh [hunk 1]
```diff
@@ -264,6 +264,12 @@ function DM_RECOVER_LOCK_CASE() {
 	# TrySync tb2: joined(a,c); tb1(a,c); tb2(a,b)
 	restart_master
 
+	# because we disabled auto-resume feature to satisfy other tests, we manually resume once to handle task was paused by
```
Comment (glorv) [inline tests/shardddl1/run.sh:267]: Shall we check the task is stopped first?

Comment (lance6716) [reply tests/shardddl1/run.sh:267]: resume-task will also return `result: true` when task is not paused

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**